src/app.py

```diff
@@ -18,4 +18,4 @@ async def lifespan(app: FastAPI):
 app = FastAPI(lifespan=lifespan)
 app.add_middleware(GZipMiddleware, minimum_size=100)
 app.include_router(scene_router.router)
-app.include_router(server_router.router)
```

Comment:
https://github.com/group-project-gut/lynx-scene-host-python/pull/26/files#r1322057532

Reply:
We are adding a new line here because it is good practice ;)